bundles/org.openhab.io.hueemulation/src/main/resources/ESH-INF/automation/moduletypes/AbsoluteDateTimeTrigger.json

```diff
@@ -0,0 +1,36 @@
+{  
```

Comment:
What is the goal for this production or test purpose?

Reply:
I'm not sure what you mean.
production is the goal of course, but how does it correlate with this file?

For the hue service I need those automation module types. At some point I like to move them to the core, but I like to discuss that with @openhab-5iver first.

Reply:
I was not aware of there goal I assumed it was some kind of test/mock response 🙄

Reply:
Ah ok. The automation module type json files are like the config description xml files. They are only really required for UI to know about inputs, outputs and config parameters.
The module originally came from Bosch I think and they decided to use json instead of the existing xml schemas.

Reply:
Yes, I do remember the chat we had about this while walking near the station of Ludwigsburg 😆